programming/sr/vision/index.md

```diff
@@ -171,62 +186,98 @@ One (possibly both) of them may change to resolve this.
 
 An `Orientation` object describes the orientation of a marker.
 
-![A visual representation of how the orientation axes work. Source: SourceBots]({{ site.baseurl }}/images/content/vision/yaw-pitch-roll.png)
-
-pitch
-:   Rotation of the marker about the cartesian x-axis, in radians.
+rot_x
+:   Rotation of the marker about the cartesian x-axis, in radians. This is a
+    pitch-like rotation.
 
-    Leaning a marker away from the camera increases the value of `rot_x`, while
-    leaning it towards the camera decreases it. A value of 0 indicates that the
-    marker is upright.
+    Leaning a marker towards the camera increases the value of `rot_x`, while
+    leaning it away from the camera decreases it. A value of either π or -π
+    indicates that the marker is upright (there is a discontinuity in the value
+    at π and -π, as both values represent the same position).
 
-yaw
-:   Rotation of the marker about the cartesian y-axis, in radians.
+rot_y
+:   Rotation of the marker about the cartesian y-axis, in radians. This is a
+    yaw-like rotation.
 
-    Turning a marker clockwise (as viewed from above) increases the value of
-    `rot_y`, while turning it anticlockwise decreases it. A value of 0 means
+    Turning a marker clockwise (as viewed from above) decreases the value of
+    `rot_y`, while turning it anticlockwise increases it. A value of 0 means
     that the marker is perpendicular to the line of sight of the camera.
 
-roll
-:   Rotation of the marker about the cartesian z-axis, in radians.
+rot_z
+:   Rotation of the marker about the cartesian z-axis, in radians. This is a
+    roll-like rotation.
 
     Turning a marker anticlockwise (as viewed from the camera) increases the
     value of `rot_z`, while turning it clockwise decreases it. A value of 0
     indicates that the marker is upright.
 
-rot_x
-:   An alias for `pitch`.
+There are additional attributes for the [principal axis rotations](https://en.wikipedia.org/wiki/Aircraft_principal_axes) of the marker.
 
-rot_y
-:   An alias for `yaw`.
+yaw
+:   A rotation about the about the vertical axis, in radians (an axis top to
+    bottom through the token). Turning a marker clockwise (as viewed from above)
+    increases the value of `yaw`, while turning it anticlockwise decreases it. A
+    value of 0 means that the marker is perpendicular to the line of sight of
+    the camera.
 
-rot_z
-:   An alias for `roll`.
+    This differs from `rot_y` in the direction that increases the value.
```

Comment:
Markdown looks to be converting this paragraph into a code block. Same for the extra paragraphs on the two below.

Reply:
![Screenshot 2023-02-08 at 22 34 30](https://user-images.githubusercontent.com/10937272/217666771-2c8290f5-864d-4c94-b75b-8661daee4aba.png)

Reply:
I think that's a function of GitHub's rendering. Here's what it looks like under Jekyll:
![image](https://user-images.githubusercontent.com/336212/217667553-bcd26b7d-4772-41d9-86c8-70404bff19db.png)